commands-handler.sh

```diff
@@ -37,6 +37,14 @@ case "$1" in
     "--disable-double-line") fancygit_config_save "double_line" "false";;
     "--enable-rich-notification") fancygit_config_save "show_rich_notification" "true";;
     "--disable-rich-notification") fancygit_config_save "show_rich_notification" "false";;
+    "--enable-tag-name") fancygit_config_save "show_tag_name" "true";;
+    "--disable-tag-name") fancygit_config_save "show_tag_name" "false";;
```

Comment:
I'm not sure if disabling/enabling tag name is interesting here. I mean, maybe wee need the tag name to be always visible, once we checkout to it. What do you think of it?